### PR TITLE
Modify concentrator algorithms

### PR DESCRIPTION
**Pull request by snwebb**
Add equal share, one bit fraction and super trigger cell 4,4,4,4 (FixedDataSizePerHGCROC)

### CODE REVIEW

**Comment by jbsauvan** [comment]
It's also needed to add the concentrator algos in the chains after they are registered

**Comment by snwebb** [comment]
done

File: produce_ntuple_multialgos_reduced_genmatch_v9_cfg.py [hunk 1]
```diff
@@ -78,6 +78,9 @@
 chains.register_concentrator("Supertriggercell", concentrator.create_supertriggercell)
 chains.register_concentrator("Threshold", concentrator.create_threshold)
 chains.register_concentrator("Bestchoice", concentrator.create_bestchoice)
+chains.register_concentrator("Supertriggercell4444Fixed",  lambda p,i : concentrator.create_supertriggercell(p,i,stcSize=cms.vuint32(4,8,8,8),fixedDataSizePerHGCROC=True ))
+chains.register_concentrator("Equalshare", concentrator.create_supertriggercell(p,i,stcSize=cms.vuint32(4,8,8,8),fixedDataSizePerHGCROC=True,type_energy_division='equalShare')))
```
Comment (jbsauvan) [inline produce_ntuple_multialgos_reduced_genmatch_v9_cfg.py]: One closing parenthesis should be removed

Comment (jbsauvan) [reply produce_ntuple_multialgos_reduced_genmatch_v9_cfg.py]: And missing lambda

File: produce_ntuple_multialgos_reduced_genmatch_v9_cfg.py [hunk 2]
```diff
@@ -78,6 +78,9 @@
...
+chains.register_concentrator("Onebit",  lambda p,i : concentrator.create_onebitfraction)
```
Comment (jbsauvan) [inline produce_ntuple_multialgos_reduced_genmatch_v9_cfg.py]: Remove lambda

**Comment by snwebb** [comment]
thanks, done.

**Pull request merged**